pages/documentation/grid/formats/cim-cgmes.md

```diff
@@ -152,17 +170,134 @@ For each `SvInjection` in the CGMES network model a new PowSyBl `Load` with attr
 
 The mapping of a CGMES `EquivalentInjection` depends on its location relative to the boundary area.
 
-If the `EquivalentInjection` is outside the boundary area it will be mapped to a PowSyBl `Generator`.
+If the `EquivalentInjection` is outside the boundary area, it will be mapped to a PowSyBl `Generator`.
 
-If the `EquivalentInjection` is at the boundary area its regulating voltage data will be mapped to the generation data inside the PowSyBl `DanglingLine` created at the boundary point and its values for `P`, `Q` will be used to define the DanglingLine `P0`, `Q0`.
+If the `EquivalentInjection` is at the boundary area, its regulating voltage data will be mapped to the generation data inside the PowSyBl `DanglingLine` created at the boundary point and its values for `P`, `Q` will be used to define the DanglingLine `P0`, `Q0`.
 
-The PowSyBl generator attributes:
+The PowSyBl generator's or dangling line's generation's attributes are created as such:
```

Comment:
The PowSyBl generators or dangling lines attributes are created as such? I am not sure I understand the sentence

Reply:
Yes, a lot of possessive forms here! I wanted to say: `Attributes of the PowSyBl generator or of the PowSyBl dangling line's generation are created as such`

I will rephrase it